app/ui/browser-modern/views/browser/tabbar/tab-pointer-area.jsx

```diff
@@ -44,11 +44,14 @@ class TabPointerArea extends Component {
 
   render() {
     return (
+      /* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Uber nit: this comment is unbalanced (one is inside of the paren and the other afterwards)